internal/transport/transport.go

```diff
@@ -726,7 +726,7 @@ type ServerTransport interface {
 	RemoteAddr() net.Addr
 
 	// Drain notifies the client this ServerTransport stops accepting new RPCs.
-	Drain()
+	Drain(debugData ...string)
```

Comment:
Why do we need a varargs here?

Reply:
@easwars 
I chose this approach for backwards compatibility to avoid breaking codes already using `Drain()` or implementing `ServerTransport` with `Drain()`. But I just checked and it seems they are just very few. I will then just use `Drain(debugData string)` instead of a variadic function.